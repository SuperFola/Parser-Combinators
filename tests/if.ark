(if 1 2 3)
(if
1
2
3)
( if 
    "a"
        1
            2)
(if 1 2)
