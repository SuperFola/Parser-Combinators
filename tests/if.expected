( Keyword:if Number:1 Number:2 Number:3 )
( Keyword:if Number:1 Number:2 Number:3 )
( Keyword:if String:a Number:1 Number:2 )
( Keyword:if Number:1 Number:2 )
( Keyword:if Number:3 Symbol:nil )
( Keyword:if ( Symbol:func Symbol:a Symbol:b ) Symbol:a Symbol:b )
( Keyword:if ( Symbol:a Symbol:b Symbol:c ) ( Symbol:d Symbol:e ) ( Symbol:f ) )